example: add one with five different layers

zarov opened the pull request:
The goal is to provide a (not so ugly) usecase with sufficient layers to test the behavior of itowns with several layers, while charging and displaying them.

There also is a sixth layer commented, that can be added to push the number of texture further than 16, and thus test some issue such as #583 and #584.

Layers added are:

- Administrative delimitations (France)
- Roads (France)
- Railways (France)
- Cadastral plan (France)
- Denomination of locations (town, lake, mountain etc...)

![multiple_layers](https://user-images.githubusercontent.com/741255/33773868-a8c96f92-dc39-11e7-90b0-4cc3e9df52e5.png)

autra reviewed (approved):
I'm fine with it, but I have a slight preference for having all the layers in `index.html`, with `.visible = false` for most of them. This won't add loading time, while still allowing us to test them when needed. I see the `index.html` as the "page for itowns developer".

WDYT?

autra commented:
To elaborate: this example doesn't add something new to already existing examples, that's why I'm advocating to put them in `index.html`.

zarov commented:
Yes, it is fine with me to remove this example and put the layers in the global `index.html`. I updated that.

autra commented:
Thanks! This Pr is already useful: it exhibits a bug in subdivision. I'm opening an issue after merging this.

autra commented on examples/layers/JSONLayers/Cada.json:
```diff
@@ -0,0 +1,164 @@
+  },
+  "version": "1.0.0",
+  "minScaleDenominator": 533.1823959624461,
+  "maxScaleDenominator": 17471320.75089743
```
What are those 2 properties for? (They are not used in itowns)

zarov replied:
I honestly don't know, I just copy/paste them :D I'll remove them.

autra commented on examples/layers/JSONLayers/Denomination.json:
```diff
@@ -0,0 +1,27 @@
+  "protocol": "wmts",
+  "id": "Denomination",
+  "title": "Geographical denomination in France",
+  "visible": false,
```
apparently this doesn't work with the current code :-( Let's remove these `visible: false`. Actually, because they were all visible, I discovered a bug, so maybe it's actually better to have them all! (sorry to backtrack on that)

zarov replied:
From what I've seen, they are not visible, but the dat.gui panel is not showing the correct status. What is the bug you discovered ? It sometimes doesn't load correctly the tiles for my part, but that's not always the case so I don't really know.

zarov replied:
I pushed a fix for the GUITools in #598

autra reviewed (commented):
Some nits though:

zarov committed:
example: complete the index one with five different layers

The goal is to provide a usecase with sufficient layers to test the
behavior of itowns with several layers, while charging and displaying
them. This differs from current layers as we can all see them at the
same time without tweaking the opacity of a layer.

autra commented:
LGTM :-)

zarov commented:
Thanks !
